.travis.yml

```diff
@@ -1,29 +1,31 @@
 language: php
 
+dist: trusty
+
 php:
-  - 5.3
   - 5.4
   - 5.5
   - 5.6
```

Comment:
Thanks for the comment.  I've raised https://github.com/whiteoctober/Pagerfanta/issues/237 for this, as unfortunately it looks like some additional work will be required to add them in.